hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -71,6 +71,22 @@ The following is a list of common features and settings:
 * **ral**: This is the instance to the auto-generated RAL model that is
   extended from `dv_base_reg_block`. In the base class, this is created using
   the RAL_T class parameter which the extended IP env cfg class sets correctly.
+* **tl_intg_alert_name**: Name of the alert that will be triggered on  TLUL
+  integrity error detection. The default name used for this type of alert is
+  "fatal_fault". The block may use a different name too - in that case, please
+  update this member to reflect the correct name in the `initialize()` method.
```

Comment:
```suggestion
  "fatal_fault". Update this variable in the 'initialize() method to change the block's alert name.
```

Reply:
prefer to clearly say to update it when the name is different from default.